pkg/secrets/pushtofile/secret_group.go

```diff
@@ -137,6 +139,12 @@ func (sg *SecretGroup) absoluteFilePath(secretsBasePath string) (string, error)
 		)
 	}
 
+	// Filename cannot be longer than allowed by the filesystem
+	_, filename := path.Split(absoluteFilePath)
+	if len(filename) > maxFilenameLen {
+		return "", fmt.Errorf("the filename '%s' is too long", filename)
```

Comment:
We've been trying to standardise on letting the string formats handle the quotes with something like
```
"the filename %q is too long"
```

Reply:
Thinking to update the messaging to be more specific about the constraint, and to provide details about the secret group that caused the failure
```
"filename %q for provided filepath for secret group %q must not be longer than %d"
```